pulp_ansible/app/settings.py

```diff
@@ -24,3 +24,5 @@
 ANSIBLE_CONTENT_HOSTNAME = settings.CONTENT_ORIGIN + "/pulp/content"
 ANSIBLE_SIGNING_TASK_LIMITER = 10
 ANSIBLE_CERTS_DIR = "/etc/pulp/certs/"
+ANSIBLE_DEFAULT_DISTRIBUTION_PATH = None
+ANSIBLE_URL_NAMESPACE = ""
```

Comment:
We need to have the docs updated with these settings prior to merging. https://docs.pulpproject.org/pulp_ansible/settings.html https://github.com/pulp/pulp_ansible/blob/main/docs/settings.rst